src/Acuminator/Acuminator.Tests/Tests/Utilities/AttributeInformation/AttributeInformationTests.cs

```diff
@@ -137,6 +136,14 @@ private async Task IsBoundAttributeAsync(string source, List<BoundType> expected
 								  new List<bool> { false, false, true },
 								  new string[] { externalAttribute1, externalAttribute2 });
 
+		[Theory]
+		[EmbeddedFileData(@"PropertyIsDBBoundFieldInheritedFromPXObjects.cs")]
+		public Task IsDBFieldFromPXObjectsAsync(string source) =>
+			IsDBFieldPropertyAsync(source,
+									expected: new List<bool> {true, false, true, false},
+									code: new string[] { });
```

Comment:
I think it is safe to remove this line, am I correct?

Reply:
Removed.